src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,377 @@
+* wallet: the holder of funds for the `sender` (may or may not be custodied).
+* Redirect URL: The URL that is the result of the PayID Easy Checkout protocol, and which can be used to redirect a client to a wallet corresponding to a particular PayID.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][] and [RFC9174][].
+
+# Introduction
+
+The PayID Easy Checkout Protocol is a minimal protocol designed to provide a standard method for online merchants to
+request a payment from a customer using only the customer's PayID. Implementations
+of the protocol should require little to no server-side engineering efforts, while creating an improved and uniform
+user experience for customers.
+
+The main focus of the Protocol is on PayID Easy Checkout Discovery, which defines how a PayID client can use a PayID
+to retrieve a PayID Easy Checkout URI Template which, when expanded, constitutes a PayID Easy Checkout URL 
+representing a resource that the customer's digital wallet can use to initiate a payment to the merchant. 
+
+Though section (TODO: link to appendix example usage section) of this specification provides an example usage of a 
+PayID Easy Checkout URL using Web Redirects, supplemental RFCs are needed to define the different ways in which a PayID
+client can utilize a PayID Easy Checkout URL. 
+
+# PayID Easy Checkout Protocol
+The PayID Easy Checkout Protocol can be used to facilitate an end-to-end checkout flow between a recipient client, such
+as an online merchant UI, and a wallet client.
+
+The protocol is comprised of two parts:
+
+1. PayID Easy Checkout Discovery
+2. Assembly of PayID Easy Checkout URL
+
+The result of the protocol is a URL, which can be used by clients to complete a payment with minimal user interaction.
+However, the PayID Easy Checkout protocol does not define how a PayID Easy Checkout URL can or should be used. Instead,
+we defer those specifications to future RFCs. 
+
+## PayID Easy Checkout Discovery
+PayID Easy Checkout extends [PAYID-DISCOVERY][] by defining a new link in the PayID metadata JRD
+returned by a PayID Discovery query. This link, defined in section (TODO: link to jrd section) 
+of this specification, includes the PayID Easy Checkout URI Template representing a resource on the wallet which can
+be used to complete a payment.
+
+E-commerce receivers who wish to perform Easy Checkout MUST query the PayID Discovery server to obtain a PayID Easy Checkout
+URI Template. Digital wallets and PayID server operators who wish to enable PayID Easy Checkout MUST include a JRD Link
+conforming to the definition in section (TODO: link to jrd section) of this paper in all PayID Easy Checkout Discovery 
+responses.
+
+E-commerce receivers SHOULD implement fallback measures to complete checkout if a user's wallet does not support PayID Easy Checkout.
```

Comment:
Should this be less specific than e-commerce? E.g., what about donations or other use-cases?